python/guide/module/gluon/parameters.md

```diff
@@ -68,13 +73,13 @@ print(net[0].collect_params())
 print(net.collect_params())
 ```
 
-This provides us with a third way of accessing the parameters of the network. If we wanted to get the value of the bias term of the second layer we could simply use this:
+This provides us with the third way of accessing the parameters of the network. If we want to get the value of the bias term of the second layer we could simply use this:
 
 ```{.python .input  n=7}
 net.collect_params()['dense1_bias'].data()
 ```
 
-Throughout the book we'll see how various blocks name their subblocks (Sequential simply numbers them). This makes it very convenient to use regular expressions to filter out the required parameters.
+By adding a regular expression as an argument to `collect_params` method, we can select only a particular set of parameters whose names are matched by the regular expression.
 
 ```{.python .input  n=8}
 print(net.collect_params('.*weight'))
```

Comment:
2 spaces between function definitions for PEP8. block1 and block2 further down. Couldn't comment on that line, sorry!